pkg/pillar/types/zedroutertypes.go

```diff
@@ -1132,6 +1132,7 @@ type NetworkPortConfig struct {
 	ProxyConfig
 	L2LinkConfig
 	WirelessCfg WirelessConfig
+	Vfs         VFList // Used only for PF
```

Comment:
This is not currently used right? (I mean info about VFs inside NetworkPortConfig) I wonder if this is every going to be needed. Likely not unless we want to use VFs as ports for management, which does not make much sense. I would not put it there unless/until there is a need for it.

Reply:
@milan-zededa We might very well have devices where the management port(s) are VFs. Depends on how many ports there are on the device and how users want to cable up the Ethernet connectivity.

But it isn't completely clear to me what content we'd have in the different data structures in such a case. So maybe we should defer working out the details?

Reply:
OK, but I think in that case there is going to be a separate NetworkPortConfig for every mgmt VF. So I would agree with "we should defer working out the details", i.e. removing this attribute until/unless needed.

Reply:
Removed

Reply:
I think it is still there or I need my eyes checked :)

Reply:
Don't rush it to the doctor, @milan-zededa :) Forgot to push the changes